lte/gateway/c/core/oai/tasks/nas/api/mme/mme_api.hpp

```diff
@@ -191,8 +191,14 @@ bool mme_ue_context_get_ue_sgs_neaf(mme_ue_s1ap_id_t mme_ue_s1ap_id);
 }
 #endif
 
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
This fun() need not be under extern C as it's definition and callee are in cpp files